.golangci.yml

```diff
@@ -6,7 +6,6 @@ run:
 linters:
   disable-all: true
   enable:
-    - bodyclose
```

Comment:
What does this linter do again?

Reply:
nothing because it is incompatible with go 1.18

Reply:
but if it did work, it would check to ensure that response bodies are closed:

https://github.com/timakin/bodyclose